packages/react-chat/src/components/Image/Background.tsx

```diff
@@ -9,6 +9,19 @@ export const BackgroundImageBase = styled(tag('div', 'background'), {
   backgroundPosition: 'center',
   backgroundRepeat: 'no-repeat',
   backgroundSize: 'cover',
+  variants: {
+    mode: {
+      bubble: {
+        width: 248,
+      },
+      embedded: {
+        width: '100%',
+      },
+    },
+  },
+  defaultVariants: {
+    mode: 'bubble',
+  },
```

Comment:
I don't think this is actually used anywhere at all, and I'm not sure why we would need variants for this component.

We could probably delete this whole component for tech debt, but removing this diff is sufficient for this PR

Reply:
this component wasn't used anywhere anyway, so I simplified `Image` in general